### DISCUSSION TITLE
Multi-page PDF with an unknown number of pages with images

### DISCUSSION THREAD

**Discussion by qertis** [pdfme/pdfme#158] [General]
I am using the jsPDF library to generate a multi-page PDF based on images. I am also using your library pdfme/generator to generate a PDF from text. You can find my code here [My code](https://github.com/gotois/archive/blob/master/src/helpers/pdfHelper.ts) .

I would like to use just one library, and pdfme/generator seems like the perfect choice. However, I couldn't find in your documentation how to create a multi-page PDF with an unknown number of pages.

Could you provide a clear example of how to use it for this purpose?

**Comment by mikmikmik** [comment]
Your link is broken

**Comment by qertis** [comment]
> Your link is broken

[Link updated](https://github.com/gotois/archive/blob/master/src/helpers/pdfHelper.ts)
